apps/server/Dockerfile

```diff
@@ -1,22 +1,34 @@
-FROM oven/bun
+FROM oven/bun AS deps
+WORKDIR /app
+
+# Copy only package.json and bun.lockb (if it exists)
+COPY package.json bun.lockb* ./
```

Comment:
We are in a monorepo, don't we need the other package.json from subfolder too ?

Reply:
Oh my bad, I forgot it's not a universal package.json😅